src/table/settings/economy_settings.ini

```diff
@@ -79,6 +79,13 @@ strval   = STR_CONFIG_SETTING_TOWN_FOUNDING_FORBIDDEN
 post_cb  = TownFoundingChanged
 cat      = SC_BASIC
 
+[SDT_BOOL]
+var      = economy.place_houses
```

Comment:
Doesn't really feel like an economy setting to me.

Reply:
I figured it belongs with `economy.found_town` (which is also not an economy setting).

Someday™️ I'd like to reorganize all the setting tables...